.expeditor/config.yml

```diff
@@ -2,56 +2,37 @@ project:
   alias: omnitruck
 
 slack:
-  notify_channel:
-    - ops-notify
-    - releng-notify
-    - omnitruck
-
+  notify_channel: releng-notify
+  
 github:
   delete_branch_on_merge: true
+  major_bump_labels:
+    - "Expeditor: Bump Version Major"
+  minor_bump_labels:
+    - "Expeditor: Bump Version Minor"
 
-# These are our Buildkite pipelines where deploys take place
-pipelines:
-  - verify:
-      description: Pull Request validation tests
-  - habitat/build:
-      description: Build and publish Habitat artifacts and export Docker containers
-  - deploy/acceptance:
-      description: Deploy changes to kubernetes.chef.co pods
-      definition: .expeditor/deploy.pipeline.yml
-      env:
-        - ENVIRONMENT: acceptance
-        - APP: omnitruck
-  - deploy/production:
-      description: Deploy changes to kubernetes.chef.co pods
-      definition: .expeditor/deploy.pipeline.yml
-      env:
-        - ENVIRONMENT: production
-        - APP: omnitruck
-
-# These actions are taken when `/expeditor promote` is run from Slack
-promote:
-  actions:
-    - built_in:promote_habitat_packages
-    - trigger_pipeline:deploy/production:
-        only_if_conditions:
-          - value_one: "{{target_channel}}"
-            operator: equals
-            value_two: stable
-  channels:
-    - acceptance
-    - stable
 
-merge_actions:
-  - trigger_pipeline:habitat/build:
-      post_commit: true
-      ignore_labels:
-        - "Habitat: Skip Build"
-        - "Expeditor: Skip All"
+# These are our Buildkite pipelines where deploys take place
+pipelines: docker/build
 
 subscriptions:
-  # These actions are taken, in order they are specified, after the Habitat packages are successfully built
-  - workload: buildkite_hab_build_group_published:{{agent_id}}:*
+  - workload: pull_request_merged:{{agent_id}}:*
```

Comment:
please use `{{github_repo}}:{{release_branch}}` instead of `{{agent_id}}`

Reply:
done